### ISSUE TITLE
consider profile dependencies in maven graph

### ISSUE THREAD

**Issue by shanon84** [khalilou88/jnxplus#1160]
Hey,
I have another issue:
let say you have a maven pom.xml with a block like this:

```
<profiles>
	<profile>
		<id>bla</id>
		<activation>
			<activeByDefault>true</activeByDefault>
		</activation>
		<dependencies>
			<dependency>
				<groupId>com.example</groupId>
				<artifactId>exampling</artifactId>
				<version>${project.version}</version>
			</dependency>
		</dependencies>
	</profile>
</profiles>
```
  
  It would be nice to also consider these dependencies in the dependency graph, especially if this profile is active while running.
  I would not expect to completely handle profiles by active status, but always considiring them to have the dependencies build for example.
  
  Best regards

**Comment by khalilou88** [comment]
Hi @shanon84,

I added profile dependencies in version 1.6.0-0 for active and not active profiles. Maybe we can make the feature smarter in the future if Nx allow that.

Please test it and provide feedbacks to publish a new version.

**Comment by shanon84** [comment]
@khalilou88,
it looks like it is not working. If you like to test yourself, see one of my repositories: monorepo (my-lib1 has a profile dependency)
I have the newest version in my package.json ("@jnxplus/nx-maven": "^1.6.0-0"), but it is not working. The nxx graph does not show the required dependency.

**Comment by khalilou88** [comment]
@shanon84 fixed in version 1.6.0-1

**Comment by shanon84** [comment]
@khalilou88 thank you. It is working now.
